app/services/healthcheck.rb

```diff
@@ -0,0 +1,60 @@
+require 'sidekiq/api'
+
+class Healthcheck
```

Comment:
I'd err toward the latter, mostly because at the moment it seems to fail entirely if any one of the checks breaks, so it might be a false positive to say `all: 'broken'`.